third_party/boringssl/OWNERS

```diff
@@ -0,0 +1,2 @@
+jseales
+godofredoc
```

Comment:
nit: newline at EOF

Are these really the right owners? I think @zanderso should be in here at least.